docs/orchestration/README.md

```diff
@@ -52,9 +52,21 @@ Prefect Core ships with an open-source backend and UI that automatically extends
 
 We recommend Prefect Server for local testing and environments with restricted access to cloud services.
 
-Get started by running `prefect server start` and navigate to `http://localhost:8080`.
+To use Prefect Server as the backend, run the following command to configure Prefect for local orchestration:
 
-**Note:** Prefect Server requires both Docker and Docker Compose.
+```bash
+$ prefect backend server
+``` 
+
+Please note the server requires [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/) to be running.
+
+To start the server, UI, and all required infrastructure, run:
+
+```bash
+$ prefect server start
+```
+
+When server is running, navigate to `http://localhost:8080`.
```

Comment:
```suggestion
When server is running, navigate to http://localhost:8080.
```

Might as well leave this as clickable

Reply:
or...

```suggestion
When server is running, navigate to [`http://localhost:8080`](http://localhost:8080).
```

Not sure how our site behaves

Reply:
The markdown link is correct.